build/buildconfig.yaml

```diff
@@ -2,7 +2,6 @@ kind: BuildConfig
 apiVersion: build.openshift.io/v1
 metadata:
   name: appassessment
-  namespace: bcgov
```

Comment:
why do we want to remove the namespace here? this can be the namespace you are building image, forexample: `e595b8-tools`

Reply:
Patrick said to remove it, and I defer to him in all things.